components/layout.tsx

```diff
@@ -63,11 +64,11 @@ export default function Layout({ title, description, children }: LayoutParams) {
         <hr />
         <footer className="footer">
           <div className="text-center">
-            <a className="footer-icon" href="https://www.instagram.com/unics_gamedev/"><Instagram style={{ fontSize: '2em' }} /></a>
-            <a className="footer-icon" href="https://www.facebook.com/UniCsGameDev/"><Facebook style={{ fontSize: '2em' }} /></a>
-            <a className="footer-icon" href="https://gitlab.cs.man.ac.uk/unics-game-development/"><Gitlab style={{ fontSize: '2em' }} /></a>
+            <a className="footer-icon" href="https://www.instagram.com/unics_uom/"><Instagram style={{ fontSize: '2em' }} /></a>
+            <a className="footer-icon" href="https://www.facebook.com/unicsmanchester/"><Facebook style={{ fontSize: '2em' }} /></a>
+            <a className="footer-icon" href="https://github.com/UniCS-GameDev/"><Gitlab style={{ fontSize: '2em' }} /></a>
```

Comment:
under `components/icons.tsx`, rename Gitlab to Github and change the svg to the github icon. Then this can be replaced